### DISCUSSION TITLE
How can I get Customized colors in readme ?

### DISCUSSION THREAD

**Discussion by parmeshwar1820** [anuraghazra/github-readme-stats#2069] [Q&A] [answered]
How can I get Customized colors in readme, especially in trophies and other stats instead of default white

**Comment by DURGESH716** [comment, marked as answer]
Heyy @parmeshwar1820 
you can refer to this link here for trophies customization: [https://github.com/ryo-ma/github-profile-trophy](Trophies), and for customized  stats you can refer @anuraghazra this one repository [https://github.com/anuraghazra/github-readme-stats](stats)
